### DISCUSSION TITLE
Nova Model transcribe issues

### DISCUSSION THREAD

**Discussion by JarVonDigital** [deepgram/community#121] [General help] [answered]
Theres an issues with transcription using the Nova model.
The start and end timestamp are off.

I ran the test on the same audio file using the different models.

**Comment by matism** [comment]
I'm having the same issue. Compared to the english enhanced model it's a difference of about 3 seconds.

**Comment by SandraRodgers** [comment, marked as answer]
@JarVonDigital @matism 

I'm happy to say that the timestamps problem you reported has now been fixed. 

FYI - There was a bug that our engineering team had fixed, but the actual fix did not get pushed to production. Customers reported problems, and the team discovered that the fix still needed to be pushed out.

Please try again and let me know if you are seeing the same issues! (Or any other issues, for that matter!)

Sandra

**Reply by matism** [reply]
Thanks, it's working now!

**Reply by JarVonDigital** [reply]
Wonderful! Really love the API!!! Thank you all for the hard work.

**Reply by SandraRodgers** [reply]
Awesome! So glad to hear that!!
